src/test/shared/telemetry/telemetryUtils.test.ts

```diff
@@ -171,14 +177,18 @@ describe('telemetryUtils', () => {
                             assert.notStrictEqual(mockService.lastEvent, undefined)
                             assert.notStrictEqual(mockService.lastEvent!.createTime, undefined)
                             assert.notStrictEqual(mockService.lastEvent!.data, undefined)
-                            assert.notStrictEqual(mockService.lastEvent!.data![0].metadata, undefined)
-                            assert.notStrictEqual(mockService.lastEvent!.data![0].metadata!.get('duration'), undefined)
+                            assert.notStrictEqual(
+                                mockService.lastEvent!.data![0].Metadata!.find(item => item.Key === 'duration')?.Value,
```

Comment:
in case `data` is empty, would it make more sense to use "data![0]?.metadata!"

Reply:
I'm surprised this wasn't flagged (probably because ?. is pretty new), fixed